generics/messageConstants/common.js

```diff
@@ -18,5 +18,9 @@ module.exports = {
     PUBLISHED : "published",
     ACTIVE_STATUS : "active",
     STARTED : "started",
-    LEAD_ASSESSOR : "LEAD_ASSESSOR"
+    LEAD_ASSESSOR : "LEAD_ASSESSOR",
+    MANUAL_RATING : "manual",
+    AUTO_RATING : "auto",
+    SUBMISSION_STATUS : "ratingPending",
```

Comment:
@deepakr123 - Add constants for all submission status

Reply:
resolved